### PR TITLE
docs: docs improvements

### PR DESCRIPTION
**Pull request by samsja**
Small improvements on the current documentation

Mainly
* use jina export (yaml to yaml) for k8s and docker compose tutorials
* remove how to in every title for better readability
* re add the containerize Executor section

### CODE REVIEW

File: docs/how-to/monitoring.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 (monitoring)=
-# How to add monitoring to Jina with Kubernetes
+# Use monitoring with Jina with Kubernetes
```
Comment (JoanFM) [inline docs/how-to/monitoring.md:2]:
```suggestion
# Use monitoring with Jina in Kubernetes
```

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: docs/fundamentals/executor/containerize-executor.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+```dockerfile
+FROM jinaai/jina:3-py37-perf
+```
+
+- Install Jina like any other Python package. You can do this by specifying Jina in the `requirements.txt`, 
+or by including the `pip install jina` command as part of the image building process.  
+
+```dockerfile
+RUN pip install jina==3
```
Comment (JoanFM) [inline docs/fundamentals/executor/containerize-executor.md:38]: maybe we can free the `version` here?

**Review by JoanFM — CHANGES_REQUESTED** [review]
I like a lot the changes, some small comments though

File: docs/fundamentals/executor/containerize-executor.md (hunk 2 above)
Comment (samsja) [reply docs/fundamentals/executor/containerize-executor.md:38]: agree

**Pull request merged**